mon-pix/app/routes/board.js

```diff
@@ -2,25 +2,26 @@ import { inject as service } from '@ember/service';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 import BaseRoute from 'mon-pix/routes/base-route';
 import RSVP from 'rsvp';
-import ENV from 'mon-pix/config/environment';
 
 export default BaseRoute.extend(AuthenticatedRouteMixin, {
-
   session: service(),
 
   model() {
-    return this.get('store').findRecord('user', this.get('session.data.authenticated.userId'))
+    return this.store.findRecord('user', this.get('session.data.authenticated.userId'))
```

Comment:
Question: c'est quoi la différence entre le this.get('store') et le this.store dans Ember ?

Reply:
Le store est directement accessible dans les routes et controllers donc pas besoin d'utiliser la syntaxe lourde de get(), après non y'a pas de différence.

Reply:
On utilise partout `this.get('store')`, si on décidait de changer de style il faudrait le faire partout (et s'assurer que c'est vraiment équivalent).